ko/sql/schema/server_stmt.rst

```diff
@@ -0,0 +1,378 @@
+	   DBNAME=demodb,
+	   USER=dev1,
+	   PASSWORD='dev1-password',
+	   COMMENT='The owner of this server is dba'
+    );
+    
+   CREATE SERVER cub.srv2 (
+	   HOST='192.168.1.8',
+	   PORT=3300,
+	   DBNAME=demodb,
+	   USER=dev2,
+	   PASSWORD='dev2-password',
+	   COMMENT='The owner of this server is cub.'
+    );
+
+.. code-block:: sql
+    
+    CREATE SERVER srv1 ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER "srv 1" ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER "srv.1" ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER cub.srv1 ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    
+    CREATE SERVER "cub"."srv 2" ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER [cub].[srv.2] ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+        
+
+위의 예시에서 *"srv.1"* 과 *[cub].[srv.2]*  이름으로 생성은 실패한다.
+*cub.srv1* 의 경우 *cub* 는 사용자 이름으로 인식되고 서버명은 *srv1* 으로 인식 될 것이다.
+        
+.. note::
+
+    서버명에는 점('.')을 사용 할 수 없습니다. 따옴표나 []로 묶은 경우라고 해도 서버명에 점('.')을 사용할 수는 없습니다.
+
+
+
+RENAME SERVER
+=============
+
+**RENAME SERVER** 구문을 사용하여 서버 이름을 변경할 수 있다.
+
+::
+
+    RENAME  SERVER [owner_name.] old_server_name AS new_server_name ;
+            
+        
+*   *owner_name*: 이름을 변경할 대상 서버의 소유자 이름을 지정한다.
+*   *old_server_name*: 이름을 변경할 대상 서버의 이름을 지정한다.
+*   *new_server_name*: 새롭게 부여될 서버의 이름을 지정한다(최대 254바이트).
+
+.. note::
+
+    서버의 소유자 또는 소유 그룹의 멤버만 정보를 변경 할 수 있다. 
+    특별히 **DBA** 또는 **DBA** 의 멤버는 모든 서버 정보를 변경할 수 있다.
+    
+    변경 후에도 소유자는 동일하게 유지된다. 소유자를 변경하기 위해서는 :ref:`ALTER SERVER 구문<owner_to>`\ 을 참고한다.
+
+
+.. code-block:: sql
+
+    -- When the current account is dba
+    RENAME SERVER srv1 AS srv2;
+    RENAME SERVER dev1.srv1 AS srv3;
+
+..
+
+   위 예제가 *dba* 계정에서 수행 되었다면 *srv1* 서버의 소유자가 *cub* 였다면 변경된 *srv2* 의 소유자도 *cub* 이다.
+   또한, *srv3* 서버의 소유자는 *dev1* 으로 계속 유지 된다. 
+
+
+DROP SERVER
+===========
+
+**DROP SERVER** 구문을 이용하여 기존의 서버를 제거할 수 있다. **IF EXISTS** 절을 함께 사용하면 해당 서버가 존재하지 않더라도 에러가 발생하지 않는다.
+
+::
+
+    DROP SERVER [IF EXISTS] [owner_name.] server_name  ;
+    
+*   *owner_name*: 제거할 서버의 소유자 이름을 지정한다.
+*   *server_name*: 제거할 서버의 이름을 지정한다.
+
+
+.. code-block:: sql
+
+    DROP SERVER srv1;
+    DROP SERVER cub.srv1;
+    DROP SERVER IF EXISTS srv2;
+    
+    
+
+.. Warning::
+
+    DROP USER 구문으로 사용자를 삭제 할 때 해당 사용자가 소유하고 있는 서버가 있다면 에러 처리되고 사용자는 삭제되지 않는다.
+    먼저 사용자가 소유하고 있는 서버를 DROP SERVER 구문으로 제거한 후에 사용자 계정을 삭제해야 한다.
+        
+::
+   
+    -- When the current account is dba   
+    csql> create user cub;
+    Execute OK. (0.000371 sec) Committed.
+
+    1 command(s) successfully processed.
+    csql> create server cub.tsrv (HOST='localhost', PORT=3300, DBNAME=demdb, USER=dev1);
+    Execute OK. (0.000761 sec) Committed.
+
+    1 command(s) successfully processed.
+    csql> drop user cub;
+
+    In the command from line 1,
+
+    ERROR: Cannot drop the user who owns database objects(class/trigger/serial etc).

```

Comment:
There is no server in the list of objects in the error message.

Reply:
Correct the error message through [CBRD-24209].
Accordingly, the text of the manual has been modified in advance.

` ERROR: Cannot drop the user who owns database objects(class/trigger/serial/server etc).`